### ISSUE TITLE
crash and closed browser of applications on windows 10

### ISSUE THREAD

**Issue by rockoodi** [556556bt/HKCantonIME#32]
no matter which apps or browser are using, it will crashed and closed the on going browser or apps with just type few character.

**Comment by LancelotChanrsw** [comment]
Yes, happened to my ex wife who uses this input method. X key especially. Y or Q sometimes.
